### DISCUSSION TITLE
move external library to another user

### DISCUSSION THREAD

**Discussion by dajiahao** [immich-app/immich#6979] [General]
I imported an external library with the admin user, is there an easy way to move it to an other user

if I add the same external lib to the user all thumbnails etc have to be created again, is there a way to avoid rerun all jobs?

**Comment by schuhbacca** [comment]
For moving libraries / assets from one user to another see: https://immich.app/docs/FAQ#how-can-i-move-all-data-photos-persons-albums-from-one-user-to-another

There is no way to avoid currently when adding the same library for multiple users.

**Reply by miguelarios** [reply]
I can't find this in the FAQ anymore

**Reply by bo0tzz** [reply]
We removed it, because it didn't work well and caused corruption.

**Comment by dajiahao** [comment]
Thanks, I missed that point in the FAQ
